docs/resources/azure_generic_resource.md

```diff
@@ -0,0 +1,133 @@
+---
+title: About the azure_generic_resource Resource
+platform: azure
+---
+
+# azure_generic_resource
+
+Use the `azure_generic_resource` Inspec audit resource to test any valid Azure resource available through Azure Resource Manager. 
+
+## Availability
+
+### Installation
+
+This resource is available in the `inspec-azure` [resource pack](/inspec/glossary/#resource-pack). To use it, add the following to your `inspec.yml` in your top-level profile:
+```yaml
+depends:
+  - name: inspec-azure
+    git: https://github.com/inspec/inspec-azure.git
```

Comment:
I wonder if this should be 
```
depends:
  - name: inspec-azure
    url: https://github.com/inspec/inspec-azure/archive/x.tar.gz
supports:
  - platform: azure
```
We have had a lot of issues in the past with end-users pointing to master

Reply:
It might even be nice to link to the example in the README in all of the docs, to avoid having to maintain this example in multiple places.